src/hotspot/share/gc/g1/g1CollectionSetCandidates.cpp

```diff
@@ -95,6 +95,21 @@ void G1CollectionCandidateList::verify() {
 #endif
 
 int G1CollectionCandidateList::compare(G1CollectionSetCandidateInfo* ci1, G1CollectionSetCandidateInfo* ci2) {
```

Comment:
I would prefer if the old comparison function would be renamed as well similarly to the other, e.g. `compare_by_efficiencies`.